pint/models/pint_dd_model.py

```diff
@@ -19,48 +20,33 @@ class DDwrapper(PSRbinaryWapper):
 
     def __init__(self,):
         super(DDwrapper, self).__init__()
-        self.BinaryModelName = 'DD'
-
-        self.add_param(Parameter(name="A0",
-             units="s",
-             description="DD model aberration parameter A0",
-             parse_value=np.double),binary_param = True)
-
-        self.add_param(Parameter(name="B0",
-             units="s",
-             description="DD model aberration parameter B0",
-             parse_value=np.double),binary_param = True)
-
-        self.add_param(Parameter(name="GAMMA",
-             units="second",
-             description="Binary Einsten delay GAMMA term",
-             parse_value=np.double),binary_param = True)
-
-        self.add_param(Parameter(name="DR",
-             units="",
-             description="Relativistic deformation of the orbit",
-             parse_value=np.double),binary_param = True)
+        self.binary_model_name = 'DD'
+        self.binary_model_class = DDmodel
```

Comment:
This is only happened in pint_dd_model.py. So it is DD model specific. I believe DD model should not be the standard model.